--no-error still outputs errors

This works for me as an npm script:

```
    "efr": "eslint-find-rules --unused ~/.eslintrc -n"
```

This does not work. It outputs errors despite the --no-error flag.

```
    "efr": "eslint-find-rules --unused ~/.eslintrc --noerror"
```

I don't understand why it outputs errors anyway though.

Looks like it's not quite fixed.

The command that was run was:
eslint-find-rules --unused ~/.eslintrc --no-error

I got this:
npm ERR! Failed at the ui-lib@0.1.0 efr script 'eslint-find-rules --unused
~/.eslintrc --no-error'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ui-lib package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint-find-rules --unused ~/.eslintrc --no-error

$ npm info eslint-find-rules version
1.14.1

On Sat, Oct 29, 2016 at 5:57 PM, Sarbbottam Bandyopadhyay <
notifications@github.com> wrote:

> @mvolkmann <https://github.com/mvolkmann> can you try --no-error with the
> latest 1.14.1 <https://www.npmjs.com/package/eslint-find-rules>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sarbbottam/eslint-find-rules/issues/194#issuecomment-257120935>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAE10Dr8MkrZxPSrRxr-1NUEE3qiYEdMks5q489cgaJpZM4KkKvX>
> .
>



-- 
R. Mark Volkmann
Object Computing, Inc.

--- a/src/bin/find.js
+++ b/src/bin/find.js
@@ -25,11 +25,13 @@ var getRuleURI = require('eslint-rule-documentation')
 
 var cli = require('../lib/cli-util')
 
-var processExitCode = argv.u && (!argv.n || argv.error) ? 1 : 0
 var getRuleFinder = require('../lib/rule-finder')
 var specifiedFile = argv._[0]
 
 var ruleFinder = getRuleFinder(specifiedFile, argv.core === false)
+
+var errorOut = argv.error && !argv.n
+var processExitCode = argv.u && errorOut ? 1 : 0
 
 if (!argv.c && !argv.p && !argv.a && !argv.u) {
   console.log('no option provided, please provide a valid option') // eslint-disable-line no-console

